.github/workflows/master_dev_ci.yml

```diff
@@ -0,0 +1,38 @@
+name: Mobile-Wallet CI[Master/Dev-Branch]
+on:
+  push:
+    branches:
+      - 'dev'
+      - 'master'
+
+  pull_request:
+    branches:
+      - '*'
```

Comment:
We don't need to add all the branches here. I think adding for dev and master PRs is suffiecient. Just change this and I think we are good to go. :smile:

Reply:
But what about the hotfix branches then, should we consider build on PRs targeting 'em?

Reply:
We can add them as per the necessity, but adding for all the branches is not required. I think we can add payment-hub as well here. Add payment-hub in PR and push too. 
Also, do the rename as mentioned earlier.

Reply:
Fine!